hail/python/hailtop/aiotools/copy.py

```diff
@@ -58,7 +29,7 @@ async def copy(*,
         if 'thread_pool' not in s3_kwargs:
             s3_kwargs['thread_pool'] = thread_pool
 
-        async with RouterAsyncFS(default_scheme,
+        async with RouterAsyncFS(default_scheme='file',
```

Comment:
You mean if we delete the LocalAsyncFS class? I think we can safely assume that anywhere our code is running has some local file system.

Reply:
This will break this init method if there's no explicit local filesystems given.

```
class RouterAsyncFS(AsyncFS):
    def __init__(self, default_scheme: Optional[str], filesystems: List[AsyncFS]):
        scheme_fs = {}
        schemes = set()
        for fs in filesystems:
            for scheme in fs.schemes():
                if scheme not in schemes:
                    scheme_fs[scheme] = fs
                    schemes.add(scheme)

        if default_scheme not in schemes:
            raise ValueError(f'default scheme {default_scheme} not in set of schemes: {", ".join(schemes)}')

        self._default_scheme = default_scheme
        self._filesystems = filesystems
        self._schemes = schemes
        self._scheme_fs = scheme_fs
```

Reply:
I was looking at old code. Nevermind